media-sound/guitarix2-0.35.2.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI="5"
+
+[[ "${PV}" = "9999" ]] && inherit git-r3
+
+PYTHON_COMPAT=( python{2_7,3_{3,4}} )
+PYTHON_REQ_USE="threads(+)"
+inherit python-any-r1 waf-utils
+
+DESCRIPTION="A simple Linux Guitar Amplifier for jack with one input and two outputs"
+HOMEPAGE="http://guitarix.sourceforge.net/"
+
+RESTRICT="mirror"
+if [ "${PV}" = "9999" ]; then
+	EGIT_REPO_URI="git://git.code.sf.net/p/guitarix/git/"
+	S="${S}/trunk"
+	KEYWORDS=""
+else
+	SRC_URI="mirror://sourceforge/guitarix/guitarix/${P}.tar.xz"
+	S="${WORKDIR}/guitarix-${PV}"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+SLOT="0"
+LICENSE="GPL-2"
+
+IUSE="avahi +capture debug faust ladspa lv2 +meterbridge nls"
+
+RDEPEND="dev-cpp/eigen:3
```

Comment:
Did you check that all these packages are really needed for runtime ? 
I thought you use `boost`  only for compiling (but I might mistaken)